FpgaConfigBuilder/src/ClassicalConstruction.java

```diff
@@ -0,0 +1,29 @@
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import datastructures.ConfigParam;
+import datastructures.FpgaConfig;
+import datastructures.IpAddress;
+import datastructures.MacAddress;
+
+public class ClassicalConstruction {
```

Comment:
Contains both the algorithmic logic of director class plus the knowledge to build products (configuration parameters).